libethereum/BlockChain.cpp

```diff
@@ -206,18 +206,15 @@ bool BlockChain::open(fs::path const& _path, WithExisting _we)
     {
         if (_we == WithExisting::Kill)
         {
-            cnote << "Killing blockchain (" << chainSubPathBlocks << ") & extras ("
-                  << extrasSubPathExtras << ") databases (WithExisting::Kill).";
-            fs::remove_all(chainSubPathBlocks);
-            fs::remove_all(extrasSubPathExtras);
+            LOG(m_loggerInfo) << "Deleting all databases. This will require a resync from genesis.";
+            fs::remove_all(chainPath);
         }
 
         fs::create_directories(extrasPath);
         DEV_IGNORE_EXCEPTIONS(fs::permissions(extrasPath, fs::owner_all));
 
         auto const extrasSubPathMinor = extrasPath / fs::path("minor");
```

Comment:
Perhaps make a constant out of `"minor"` as it's used in more that one place now

Reply:
@gumb0 I don't see where it's being used multiple times in a function?

Reply:
In another function (`rebuild`)

Reply:
Ah, yes, most of the paths are used in both places. I can move them to anonymous namespace vars.